memif-attach/main.go

```diff
@@ -32,14 +33,35 @@ import (
 	"github.com/edwarnicke/vpphelper"
 )
 
+type MemifMsg struct {
+	HelloMsgType HelloMsg
+	InitMsgType InitMsg
+}
 // HelloReplyMsg type
 type HelloMsg struct {
-	max_region int16 
-	max_m2s_ring int16 
-	max_s2m_ring int16
-	min_version int16 
-	max_version int16
-	max_log2_ring_size int8 
+	Max_region uint16 
+	Max_m2s_ring uint16 
+	Max_s2m_ring uint16
+	Min_version uint16 
+	Max_version uint16
+	Max_log2_ring_size uint8 
+	Name [32]uint8
+}
+
+// InitMsg
+const MEMIF_SECRET_SIZE = 24
+type MemifInterfaceMode int
+const (
+	MEMIF_INTERFACE_MODE_ETHERNET MemifInterfaceMode = iota
+	MEMIF_INTERFACE_MODE_IP
+	MEMIF_INTERFACE_MODE_PUNT_INJECT
+)
+type InitMsg struct {
```

Comment:
What value should I put in initMsg to send to vpp master?

Reply:
No need to specify a secret at this time.